main.tf.json

```diff
@@ -1377,16 +1560,23 @@
                         "aws_iam_role.clumio_ssm_notification_role",
                         "aws_iam_role_policy.clumio_ssm_notification_policy",
                         "aws_iam_policy.clumio_s3_protect_policy",
+                        "aws_iam_policy.clumio_dynamodb_policy",
                         "aws_iam_policy.clumio_warm_protect_dynamodb_policy"
                     ],
                     "discover_version": "4.2",
+                    "properties": {
+                        "DynamoDbSecureVaultPolicyArn": "${aws_iam_policy.clumio_dynamodb_policy[0].arn}",
+                        "DynamoDbWarmProtectPolicyArn": "${aws_iam_policy.clumio_warm_protect_dynamodb_policy[0].arn}",
+                        "PermissionsBoundaryArn": "${aws_iam_policy.clumio_iam_permissions_boundary.arn}"
+                    },
                     "protect_config_version": "${var.is_protect_enabled ? \"19.2\" : \"\"}",
+                    "protect_dynamodb_version": "${var.is_protect_enabled && var.is_dynamodb_enabled ? \"2.0\" : \"\"}",
                     "protect_ebs_version": "${var.is_protect_enabled && var.is_ebs_enabled ? \"20.1\" : \"\"}",
                     "protect_ec2_mssql_version": "${var.is_protect_enabled && var.is_ec2_mssql_enabled ? \"2.1\" : \"\"}",
                     "protect_rds_version": "${var.is_protect_enabled && var.is_rds_enabled ? \"18.1\" : \"\"}",
                     "protect_s3_version": "${var.is_protect_enabled && var.is_s3_enabled ? \"2.1\" : \"\"}",
-                    "protect_warm_tier_dynamodb_version": "${var.is_protect_enabled && var.is_warmtier_enabled && var.is_warmtier_dynamodb_enabled? \"2.1\" : \"\"}",
-                    "protect_warm_tier_version": "${var.is_protect_enabled && var.is_warmtier_enabled ? \"1.1\" : \"\"}",
+                    "protect_warm_tier_dynamodb_version": "${var.is_protect_enabled && var.is_dynamodb_enabled? \"2.1\" : \"\"}",
+                    "protect_warm_tier_version": "${var.is_protect_enabled && var.is_dynamodb_enabled ? \"1.1\" : \"\"}",
```

Comment:
hmm this actually seems like we should release a new version of the PostProcess resource as well? What does CFT do, it also tracks Cold DynamoDB support under the Warm Tier version?

Reply:
No. For Cold DynamoDB there is a separate entry:
"protect_dynamodb_version": "${var.is_protect_enabled && var.is_dynamodb_enabled ? \"2.0\" : \"\"}",

We are using the is_dynamodb_enabled  flag to determine whether to set the version for the protect_warm_tier_version and protect_warm_tier_dynamodb_version.

No change is required for this on the provider post_process resource.